Readme.md

```diff
@@ -162,13 +162,17 @@ options.
 
 ## API
 
-### Airbreak.createClient(apiKey, [env])
+### var airbrake = Airbrake.createClient(key, [env])
```

Comment:
Thanks @mmcdaris, I felt the documentation a bit lacking when I first checked out this package. Glad to know Airbrake is now directly maintaining it!